programmer-skills/start-to-develop-js.md

```diff
@@ -159,11 +158,11 @@ there:
 ![alert with your name](/images/brackets_alert.png)
 
 This is an easy way to start trying JavaScript snippets, but would be
-better if we split our html with JavaScript in two files. One for html,
-and other for JavaScript code.
+better if we split our html with JavaScript into two files. One for html,
+and another for JavaScript code.
```

Comment:
propose "the other" instead of another, given that there are only two objects being described here.